rest-api/flows-metrics.sh

```
@@ -5,6 +5,19 @@ END_DATE="${2:-}"
 STATUS="${3:-}"
 source_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 if [ -f "$source_dir/.env" ]; then export $(grep -v '^#' "$source_dir/.env" | xargs); fi
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)